packages/hello-gsm/src/PageContainer/ManualPage/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { Header } from 'components';
+import type { NextPage } from 'next';
+import * as S from 'PageContainer/InformationPage/style';
+import * as DS from 'components/InformationDescription/style';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+const ManualPageStyle = styled.div`
+  height: 100vh;
+  position: relative;
+  overflow-x: hidden;
+`;
+
+const ManualPage: NextPage = () => {
+  const location = () =>
+    (window.location.href = ' https://accounts.kakao.com/weblogin/account');
```

Comment:
```suggestion
  const location = () =>
    window.open('https://accounts.kakao.com/weblogin/account');
```

이런 식으로 하여 새 창에서 열리도록 하는건 어떨까요?

Reply:
넵 새창으로 여는게 더 좋겠네요!